modules/my_health/docs/schemas/SecureMessageSearchRequest.yml

```diff
@@ -0,0 +1,28 @@
+type: object
+properties:
+  message_id:
+    type: integer
+    example: 1112233
+  sender:
+    type: string
+    example: "Alice"
+  recipient:
+    type: string
+    example: "Bob"
+  category:
+    type: string
+    example: "COVID"
+  subject:
+    type: string
+    example: "My Test Results"
+  from_date:
+    type: string
+    description: Search for messages sent after this date
+    example: "2022-01-01"
+  to_date:
+    type: string
+    description: Search for messages sent before this date
+    example: "2022-12-31"
+  exact_match:
+    type: boolean
+    description: Search using the exact strings provided
```

Comment:
If exact_match is false, what is the behavior? Substring match? Some sort of fuzzy match?

Reply:
Updated the description to clarify the behavior